client_src/js/printerManager/printer-constructor.js

```diff
@@ -0,0 +1,323 @@
+import Validate from "../lib/functions/validate";
+import UI from "../lib/functions/ui";
+import OctoFarmClient from "../lib/octofarm_client.js";
+
+let newPrintersIndex = 0;
+
+const removeLine = function (element) {
+  element.remove();
+};
+class Printer {
```

Comment:
- Split out from this file
- Nice idea but it would be nice if the printer could be serialized easily by having a `serialize` function. This way the object's data can easily be sent back to backend
- File name would be `printer.manager.js` if we'd be managing multiple printers here.